build.gradle

```diff
@@ -42,7 +42,7 @@ jacocoTestReport {
 
 
 ext {
-    web3jVersion = '4.0.3'
+    web3jVersion = '4.0.4'
```

Comment:
Can you double check this version number?

Reply:
Yup - this is fine.